release notes/v0.34.0.md

```diff
@@ -0,0 +1,31 @@
+k6 v0.34.0 is here! :tada: It introduces the already announced `k6/execution` API and includes some more enhancements and a bunch of minor bug fixes.
+
+## New Features
+
+* k6/execution API
+
+Introduced a new module importable with the `k6/execution` path for fetching information about the execution state through a series of exported properties. It makes available information about VUs, Iterations and Scenarios.
+
+**Docs**: [grafana/xk6-execution](https://github.com/grafana/xk6-execution)
```

Comment:
I would definitely expand on this with at least 1 example and what it's useful for

Reply:
I have added your gist in a simplified version.